flang/test/Lower/intrinsic-procedures/ANINT.f90

```diff
@@ -0,0 +1,9 @@
+! RUN: bbc -emit-fir %s -o - | FileCheck %s
```

Comment:
Move all these to nint test file?

Reply:
Wouldn't that be a bit inconsistent though?

Reply:
OK